doc/user_manual/postprocessor.tex

```diff
@@ -145,6 +146,9 @@ \subsection{PostProcessor}
 %%%%%%%%%%%%%%% TSACharacterizer %%%%%%%%%%%%%%%%%%%
 \input{PostProcessors/TSACharacterizer.tex}
 
+%%%%%%%%%%%%%%% SparseSensing %%%%%%%%%%%%%%%%%%%
+\input {Postprocessors/SparseSensingdocumentation.tex}`
```

Comment:
Change Postprocessors to PostProcessors (otherwise it will not work on Linux)

Reply:
Thank you for reviewing this PR. I have just pushed updates based on these comments.